### PR TITLE
Fix for checking System Drive space

### PR DESCRIPTION
**Pull request by mlipok**
Related to: https://github.com/rcmaehl/WhyNotWin11/issues/374

### CODE REVIEW

File: WhyNotWin11.au3 [hunk 1]
```diff
@@ -61,6 +61,7 @@ Switch @OSVersion
 		;;;
 EndSwitch
 
+Global $WINDOWS_DRIVE = StringLeft(@WindowsDir,3)
```
Comment (htcfreek) [inline WhyNotWin11.au3:64]:
```suggestion
Global $WINDOWS_DRIVE = EnvGet(SystemDrive)
```

**Review by htcfreek — CHANGES_REQUESTED** [review]

File: WhyNotWin11.au3 (hunk 1 above)
Comment (mlipok) [reply WhyNotWin11.au3]: Thanks.
Just commited proper usage:

```AutoIt
Global $WINDOWS_DRIVE = EnvGet("SystemDrive")
```

**Comment by htcfreek** [comment]
Not sure if it make sense to merge it. I planned to rework this check tomorrow.

**Plan:**
Cehck: Size of Windows Partition, Disk with Windows partition and show if other internal disks would meet the requirements.

**Comment by micwoj92** [comment]
> Not sure if it make sense to merge it. 

Will wait for Robert to decide.

**Comment by htcfreek** [comment]
@micwoj92 
(A bit of offtopic.)

Would it be possible to have three text lines on check result in gui?

**Comment by micwoj92** [comment]
I mean yeah, it is possible. Currently all fields are same size, so I think that it should be done by making the font smaller. Otherwise it can look weird.

**Comment by htcfreek** [comment]
> I mean yeah, it is possible. Currently all fields are same size, so I think that it should be done by making the font smaller. Otherwise it can look weird. But the final decision is as always up to Robert.

I will try wite the first information in one line like this: `Partition "C:" = xy GB (Drive # = xy GB)`

**Comment by mlipok** [comment]
> Not sure if it make sense to merge it. I planned to rework this check tomorrow.
As so far yours ISSUE https://github.com/rcmaehl/WhyNotWin11/issues/374
Was fixed by my PR

> **Plan:**
> Cehck size of Windows Partition, size of Disk with Windows partition and show if other internal disks would meet the requirements.

You can work on your Plan in next/further PR

**Review by rcmaehl — APPROVED** [review]

**Pull request merged**